### PR TITLE
Meta module test and fix

### PR DESCRIPTION
**Pull request by r4f4**

### CODE REVIEW

File: fus.c [hunk 1]
```diff
@@ -832,6 +832,11 @@ resolve_all_solvables (Pool  *pool,
                       Id p = t.elements[j];
                       Solvable *s = pool_id2solvable (pool, p);
 
+                      /* Make sure to include the module into the pile even if
+                       * it doesn't contain any components (e.g, a meta-module)
```
Comment (igor-raits) [inline fus.c]: I would say "empty"

Comment (r4f4) [reply fus.c]: @sgallagher gave out to me today for calling it "empty".

Comment (r4f4) [reply fus.c]: I don't mind changing it to "empty" if you think it's not an ambiguous term.

Comment (igor-raits) [reply fus.c]: please make it empty

**Pull request merged**